db/seeds.rb

```diff
@@ -1,14 +1,51 @@
-40.times do |f|
-  book = Book.create!(
-    name: Faker::Name.name,
-    price: Random.rand(1..99),
-    quantity: Random.rand(1..99)
+default_password = "123456".freeze
+
+def generate_image_url
+  id = rand(1..900)
+  "https://picsum.photos/id/#{id}/600/900"
+end
+
+def generate_avatar_url
+  id = rand(1..900)
+  "https://picsum.photos/id/#{id}/600/600"
```

Comment:
Did you see, it same as `generate_image_url ` ???

Reply:
book cover 600x900 còn avatar 600x600 a ơi

Reply:
Truyền tham số vào ?

Reply:
generate_image_url 600, 900

Reply:
Book lấy ảnh chữ nhật còn avatar lấy ảnh vuông